osbs/build/build_request.py

```diff
@@ -486,7 +486,13 @@ def render_koji_promote(self, use_auth=None):
 
             if use_auth is not None:
                 self.dj.dock_json_set_arg('exit_plugins', 'koji_promote',
-                                          'use_auth', use_auth)
+                                            'use_auth', use_auth)
+
+            if self.spec.koji_use_kerberos.value:
```

Comment:
@lcarva I'm good either way, the only reasdon I added `koji_use_kerberos` was there is a `use_kerberos` config flag already for the client and I was aiming for consistency. Which ever you and @twaugh (and Co.) prefer, I'm happy to go that direction.